glfw/src/native.zig

```diff
@@ -0,0 +1,290 @@
+const std = @import("std");
+const c = @import("c.zig").c;
+
+const Window = @import("Window.zig");
+const Monitor = @import("Monitor.zig");
+const Error = @import("errors.zig").Error;
+const getError = @import("errors.zig").getError;
+
+const internal_debug = @import("internal_debug.zig");
+
+pub fn getWin32Adapter(monitor: Monitor) [*:0]const u8 {
```

Comment:
I'd also love to copy all the docs from glfw3native.h into this file as `///` doc comments on these functions, similar to what we do for others. Happy to do this boring work if you don't want to

Reply:
I will do that, dont worry, just want to figure out the actual work first